gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -917,6 +924,7 @@ accept_socket_async_cb (GObject * source, GAsyncResult * result,
       nns_loge ("Failed to create message handler thread.");
       nnstreamer_query_close (conn);
       g_free (thread_data);
+      conn->msg_thread = 0;
```

Comment:
line 925. after calling nnstreamer_query_close(), conn is freed. How about moving this into nnstreamer_query_close()?

Reply:
moved.